index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Jerry && Faith</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <div id="main">
+        <div id="title">
+          <h1>Dr Norman Borlaug</h1>
+          <h4>The man who saved a billion lives</h4>
+        </div>
+
+    </div>
+    <figure id="img-div">
+      <img id="image" src="./images/tribute-page-main-image.jpg" alt="">
+      <figcaption id="img-caption">
+        Dr. Norman Borlaug, third from the left, trains biologists in 
+        Mexico on how to increase wheat yields - part of his life-long war on hunger.
+      </figcaption>
+    </figure>
+    <div id="tribute-info">
+      <h2>Here's a time line of Dr. Borlaug's life:</h2>
+      <!-- <p><strong></strong> -  Born in Cresco, Iowa </p> -->
```

Comment:
please remove unnecessary comments from your code to make it more readable and easy to maintain

Reply:
- [ ]  your comment here sounds like you are giving an order, please ensure you sound as polite as possible when giving reviews, for example use words such as `please`, `i think it would`, `i suggest`.

- [ ] please also ensure to state the reason why you are requesting a change, that way the student understands why the change is requested and knows how to organise his/her code in a professional manner +1